code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm

```diff
@@ -583,6 +583,27 @@
 	virus_suspectibility = 0.5
 	resulting_atoms = list(/mob/living/basic/frog = 1)
 
+/datum/micro_organism/cell_line/axolotl
+	desc = "caudata amphibian cells"
+	required_reagents = list(/datum/reagent/consumable/nutriment/protein)
+
+	supplementary_reagents = list(
+		/datum/reagent/ants = 3,
+		/datum/reagent/consumable/eggwhite= 2,
+		/datum/reagent/consumable/nutriment/vitamin = 1,)
+
+	suppressive_reagents = list(
+		/datum/reagent/toxin/carpotoxin = -3,
+		/datum/reagent/toxin/bungotoxin = -3,
+		/datum/reagent/toxin/spore = -3,
+		/datum/reagent/toxin/plantbgone = -2, //GAY FROGS
```

Comment:
```suggestion
		/datum/reagent/toxin/plantbgone = -2, //GAY AXOLOTLS
```

Reply:
That's a good idea. Wasn't sure on // so I left it alone. Minor change that will surely not destroy an entire station.